projects/core/src/i18n/translatable.ts

```diff
@@ -5,11 +5,15 @@
  */
 
 export interface Translatable {
-  key?: string;
+  key?: string | string[];
   params?: TranslatableParams;
   raw?: string;
 }
 
 export interface TranslatableParams {
   [param: string]: any;
 }
+
+export function isTranslatable(input: any): input is Translatable {
+  return typeof input !== 'string' && ('key' in input || 'raw' in input);
```

Comment:
Why do we check `typeof input !== string`? in the type guard for `Translatable`?
Shouldn't we check it somewhere else (outside `isTranslatable()`)?